cartographer_ros/cartographer_ros/node.h

```diff
@@ -122,6 +124,7 @@ class Node {
   std::thread occupancy_grid_thread_;
   bool terminating_ = false GUARDED_BY(mutex_);
 
+  ::ros::Publisher trajectory_nodes_list_publisher_;
```

Comment:
Move this after `submap_list_publisher_` to keep the publishers a bit more together?